packages/jest-matchers/src/index.js

```diff
@@ -20,6 +20,13 @@ import type {
 } from 'types/Matchers';
 
 const utils = require('jest-matcher-utils');
+const extractExpectedAssertionsErrors = require('./extractExpectedAssertionsErrors');
+const {
+  getMatchers,
+  getState,
+  setMatchers,
+  setState,
```

Comment:
Since these are functions, not belonging to a class, wouldn't it be better to use this naming?
* `getMatchersState`
* `setMatchersState`